[All Hosts] (overview) Reorienting overview for the M365 ecosystem (#5537)
OfficeDev/office-js-docs-pr: docs/develop/develop-overview.md

For more information about extending the Office UI and designing the add-in's UX, see [Office UI elements for Office Add-ins](../design/interface-elements.md).

> [!NOTE]

> > It is not necessary for an add-in to have any custom UI. An add-in can consist of only a custom Copilot agent that interacts with an Office document in response to natural language instructions given in Copilot chat.

Reviewer: ```suggestion
> It isn't necessary for an add-in to have any custom UI. An add-in can consist of only a custom Copilot agent that interacts with an Office document in response to natural language instructions given in Copilot chat.
```
